Program.cs

```diff
@@ -18,8 +18,15 @@ namespace scrapysharp_dt2020
 {
     class Program
     {
+        static string LoggingPath = Directory.GetCurrentDirectory() + "\\Logs\\";
```

Comment:
All of the other paths in this project are relative, and include Path.DirectorySeparatorChar so that the program doesn't blow up when trying to compile and run on other operating systems.

Example: $"ranks{Path.DirectorySeparatorChar}joinedRanks2020.csv"